setup.py

```diff
@@ -14,8 +14,7 @@
 
 import os
 import re
-import sys
-from setuptools import setup, find_packages, Extension
```

Comment:
Heh, I'm surprised that pylint didn't catch these unused imports

Reply:
we don't run pylint on `setup.py`

Reply:
Oh I just assumed we did because we run black on it. I guess it would just be more of a nuisance than anything.

Reply:
yeah, I dunno why we don't, but for some reason we don't seem to.